src/pages/RFOX/components/Stake/StakeInput.tsx

```diff
@@ -250,7 +260,7 @@ export const StakeInput: React.FC<StakeInputProps & StakeRouteProps> = ({
       stakingAssetAccountId,
       stakingAssetId,
       stakingAmountCryptoBaseUnit: toBaseUnit(amountCryptoPrecision, stakingAsset.precision),
-      runeAddress,
+      runeAddress: currentRuneAddress ?? runeAddress,
```

Comment:
This is probably the most important

Now that we don't permit the users to change the address while staking again, we should use the already setup address from the smart contract instead of any other selected address

Reply:
This will need to be added to disabled checks too or users will be rugged trying to deposit with an active position and a custom addy

Reply:
I'm not sure to get it?

Users are not able anymore to deposit with a custom addy, they should change it to that custom addy before depositing

Reply:
Then we'll need to improve the flow to inform users, there is nothing in the UI that says so:

<img width="481" alt="image" src="https://github.com/user-attachments/assets/0a3a5720-9d0f-44a3-99eb-b73901108943">

As a user, this is broken.

What's the rationale for forcing users to re-enter their custom address? The address definitely exists in state, we should be able to fix deposit with a oneliner in `isDisabled`

Reply:
Nah this was a bug during the time you tested, it works not, the user doesn't have to enter any address to deposit for the second time or any other time if he has active staking